config/config.yaml

```diff
@@ -115,6 +115,13 @@ datasets:
       other_files: []
       # Relative path from the spras directory
       data_dir: "input"
+    
+gold_standard:
+    -
+      label: gs
```

Comment:
Will we have the same constraints on labels here in terms of special characters? Let's remember to add a comment about that as documentation and check validity when parsing.